### PR TITLE
LPD-23844 - LXC- import organization from uat to prod fails

### PR DESCRIPTION
**Pull request by matyaswollner**
https://liferay.atlassian.net/browse/LPD-23844

### CODE REVIEW

**Comment by matyaswollner** [comment]
ci:test:sf

**Comment by matyaswollner** [comment]
ci:test:sf

File: portal-impl/src/com/liferay/portal/upgrade/v7_4_x/PortalUpgradeProcessRegistryImpl.java [hunk 1]
```diff
@@ -391,6 +391,8 @@ protected String[][] getTableAndPrimaryKeyColumnNames() {
 				"PortalPreferenceValue", "key_", "VARCHAR(1024) null"));
 
 		upgradeVersionTreeMap.put(new Version(30, 1, 1), new UpgradeRelease());
+		upgradeVersionTreeMap.put(
```
Comment (smottal) [inline portal-impl/src/com/liferay/portal/upgrade/v7_4_x/PortalUpgradeProcessRegistryImpl.java:394]: new empty line before this

**Review by smottal — CHANGES_REQUESTED** [review]

**Comment by smottal** [comment]
@matyaswollner only a very little change. Then run sf again, relevant and um test suites. Thx

**Comment by matyaswollner** [comment]
ci:test:sf

**Comment by matyaswollner** [comment]
ci:test:relevant

**Comment by matyaswollner** [comment]
ci:test:um

**Comment by smottal** [comment]
ci:report:2565101775

**Comment by smottal** [comment]
ci:forward

**Pull request closed without merging**